cmd/containerd/main.go

```diff
@@ -168,14 +169,18 @@ func before(context *cli.Context) error {
 		}
 		fh.Close()
 		log.G(global).Infof("containerd default config written to %q", fh.Name())
-		os.Exit(0)
+		return nil
```

Comment:
This changes the request by @crosbymichael for `exit(0)` after write default config; see [comment here](https://github.com/containerd/containerd/pull/688#discussion_r109765254)